### ISSUE TITLE
[Nightly 72][tabs not on top] Tabs position has shifted...

### ISSUE THREAD

**Issue by nicolaasjan** [Aris-t2/CustomCSSforFx#249]
1. Describe the issue
On the current Nightly (2019-11-14) the position of the tabs has shifted.

2. What should have happened instead?
The tabs need to be lower.

3. Steps to reproduce the issue?
 Update to the latest Nightly and enable:
`@import "./css/tabs/tabs_below_navigation_toolbar_fx68.css"; /**/`

4. SCREENSHOTS with/without this projects files (drag & drop images into this post)

With this projects files:
![screenshot3](https://user-images.githubusercontent.com/14093220/68855222-18fd9600-06de-11ea-8405-7bbd14f98e65.png)

Without this projects files:
![screenshot4](https://user-images.githubusercontent.com/14093220/68855911-95dd3f80-06df-11ea-9450-a08890d7e6fb.png)

5. System information
- OS & OS version: Linux Mint 19.2 Mate
- OS theme name: Mint-X/Emerald Window Decorator
- DPI / HiDPI resolution: 
- Firefox version: 72.0a1
- Firefox theme / lw-theme: Standard
- This projects settings the issue occurs with: 
`@import "./css/tabs/tabs_below_navigation_toolbar_fx68.css"; /**/ /* <-- tabs not on top - Fx68+ - Windows/Linux */`

**Comment by Aris-t2** [comment]
Possible workaround (add code to the bottom of your userChrome.css or into my_userChrome.css):

```css
#TabsToolbar {
  bottom: -3px !important;
}
```

**Comment by nicolaasjan** [comment]
Thanks!
This works.

BTW, There are more problems with today's Nightly, like [this one](https://bugzilla.mozilla.org/show_bug.cgi?id=1596176]).

![screenshot1](https://user-images.githubusercontent.com/14093220/68889826-fb502100-071d-11ea-85d8-1e3b9653a707.png)

And on [Reddit](https://old.reddit.com/r/firefox/comments/dtulxr/weekly_nightly_discussion_for_20191109_20191115/) I saw some complaining about userChrome.css  as well.
Maybe the tab quirk will be solved by Mozilla in the near future.

**Comment by Aris-t2** [comment]
Saw this one too, but that is/was a Fx bug.

The initial problem can be solved using this code too. It appears to be more stable in my tests.

```css
#TabsToolbar {
  min-height: 0 !important;
  height: var(--tab-min-height_tnot) !important;
  max-height: unset !important;
}
```

**Comment by Aris-t2** [comment]
Reopened until the fixed code is online, so others can see the solution too.

**Comment by Jgr9** [comment]
Not working for me.

I updated the css, but cleared the userChrome and made it just this, which pretty much what my old one is as well (plus some other unrelated things) (and I also keep it from using config because hadn't needed it or didn't want it before):
`@import "./css/tabs/tabs_below_navigation_toolbar_fx65.css";

#TabsToolbar {
  bottom: -3px !important;
}`

That's all that's in the userChrome (or the other potential fix), but they aren't fixing mine.  Is there more too it now?

**Comment by Aris-t2** [comment]
@Jgr9 
Windows, Linux or macOS? I've only tested Windows for now.

**Comment by Jgr9** [comment]
Windows.

**Comment by Aris-t2** [comment]
`tabs_below_navigation_toolbar_fx68.css` + this code:
```css
#TabsToolbar {
  min-height: 0 !important;
  height: var(--tab-min-height_tnot) !important;
  max-height: unset !important;
}
```

(with CustomCSSforFx default settings)
![1](https://user-images.githubusercontent.com/6013569/68971759-8dbdf680-07ea-11ea-9c35-f2b13449b1bf.png)

(without any other CustomCSSforFx settings)
![2](https://user-images.githubusercontent.com/6013569/68972088-4421db80-07eb-11ea-8415-9c20e7cb8b88.png)

**Comment by Jgr9** [comment]
Again, I have that and my screenshot is what it looks like.

**Comment by Jgr9** [comment]
You know, I guess I should be asking why this is messed up for me with no chrome whatsoever, and nobody is mentioning it on the Nightly forum.  But I just don't know....

**Comment by Aris-t2** [comment]
Test a new browser profile just to be sure.

At the current state Nightly 72 is a complete mess even without any (CSS/JS) tweaks or modifications. Mozilla managed again to heavily cripple the browser ui. Since its the "nightly" channel its meant not to work properly. After all it is a developer playground.

**Comment by Jgr9** [comment]
Well I've got my release (70?) FF set up and looking how I want it.  But once that hits around 72 (maybe later?) it's just gonna happen to that right? probably?

**Comment by nicolaasjan** [comment]
Thanks for the fix in version 2.8.6!

**Comment by Jgr9** [comment]
I'm guessing it's the code above?
